WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -97,4 +105,26 @@ public void onJSONException(JSONException exception) {
             }
         });
     }
+
+    @Override
+    public void onPersonSelected(Person person) {
+        FragmentManager fragmentManager = getFragmentManager();
+        PersonDetailFragment personDetailFragment = (PersonDetailFragment) fragmentManager.findFragmentByTag(KEY_PERSON_DETAIL_FRAGMENT);
+
+        long personID = person.getPersonID();
+        int localTableBlogID = person.getLocalTableBlogId();
+        if (personDetailFragment == null) {
+            personDetailFragment = PersonDetailFragment.newInstance(personID, localTableBlogID);
+        } else {
+            personDetailFragment.setPersonDetails(personID, localTableBlogID);
+        }
+        if (!personDetailFragment.isAdded()) {
+            FragmentTransaction fragmentTransaction = getFragmentManager().beginTransaction();
+            fragmentTransaction.setCustomAnimations(R.animator.fragment_slide_in_from_right, R.animator.fragment_slide_out_to_left,
```

Comment:
I think you can make the animation basically the same as the activity animation by using:
```
fragmentTransaction.setCustomAnimations(
                    R.animator.fragment_slide_in_from_right,
                    0,
                    0,
                    R.animator.fragment_slide_out_to_right);
```
And then using `fragmentTransaction.add` rather than `fragmentTransaction.replace`. You'll also need to set a background color for the fragment - otherwise it the activity will be visible beneath it.